_data/sites.json

```diff
@@ -13323,7 +13323,7 @@
         "name": "Walmart",
         "url": "https://www.walmart.com/account/api/ccpa-intake?native=false&app=gm&type=access",
         "difficulty": "hard",
-        "notes": "Only possible for California residents.",
+        "notes": "Select California as your state of residence, they don't check it.",
```

Comment:
I'd rather not suggest lying, people can read between the lines. If you can reword it better according to this, I'd take it.

Reply:
If you select another state it won't let you proceed.